### PR TITLE
Account Activation and Password Reset

### PR DESCRIPTION
**Pull request by Iskandar19**

### CODE REVIEW

**Comment by Iskandar19** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'password_resets/new'
```
Comment (ngocvt-0484) [inline config/routes.rb:2]: remove this routes because it defined below on line 17

**Comment by Iskandar19** [comment]
Ready

File: app/assets/stylesheets/password_resets.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the PasswordResets controller here.
```
Comment (huongnt-2545) [inline app/assets/stylesheets/password_resets.scss:1]: delete this file if it's not useful

File: app/helpers/password_resets_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module PasswordResetsHelper
```
Comment (huongnt-2545) [inline app/helpers/password_resets_helper.rb:1]: delete this file

File: app/views/users/index.html.erb [hunk 4]
```diff
@@ -1,10 +1,12 @@
-<% provide(:title, t(:all_users)) %>
-<h1><%= t(:all_users) %></h1>
+<% provide(:title, "All users") %>
+<h1>All users</h1>
 
-<%= will_paginate @users %>
```
Comment (huongnt-2545) [inline app/views/users/index.html.erb:4]: keep this, don't change

File: app/views/users/index.html.erb [hunk 5]
```diff
@@ -1,10 +1,12 @@
...
+<%= will_paginate %>
 
 <ul class="users">
-  <%= render user %>
+  <% @users.each do |user| %>
+    <%= render user %>
+  <% end %>
 </ul>
 
-<%= will_paginate @users %>
```
Comment (huongnt-2545) [inline app/views/users/index.html.erb:10]: keep this, don't change

File: test/controllers/password_resets_controller_test.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/controllers/password_resets_controller_test.rb:1]: delete this file

**Comment by Iskandar19** [comment]
ready

**Pull request merged**